mesh/accountSettings/urls.py

```diff
@@ -2,5 +2,9 @@
 from . import views as accountsettings_views
 
 urlpatterns = [
-    path("displayTheme", accountsettings_views.display_theme, name="display_theme")
+    path("displayTheme", accountsettings_views.display_theme, name="display_theme"),
+
+    # GET    /api/settings/:account_id
+    # PATCH  /api/settings/:account_id
+    path("settings/<int:account_id>", accountsettings_views.showSettings.as_view(), name="settings")
```

Comment:
Just checking, was this the path it had when we were working through those weird trailing slash issues? If the endpoint works fine then disregard this, I'm just curious.